### PR TITLE
Total Hack but it works and probably doesnt softlock

### PR DESCRIPTION
**Pull request by Zedb0T**

### CODE REVIEW

File: goal_src/jak1/levels/rolling/rolling-robber.gc [hunk 1]
```diff
@@ -59,6 +59,8 @@
   )
 
 (defbehavior fuel-cell-init-as-spline-slider fuel-cell ((arg0 handle) (arg1 float) (arg2 float) (arg3 int))
+
+     (when  (not (task-closed? (game-task rolling-robbers)(task-status need-resolution)))
```
Comment (dallmeyer) [inline goal_src/jak1/levels/rolling/rolling-robber.gc:63]: do you even need this with the check inside the rolling-robbers code? otherwise seems fine

Comment (Zedb0T) [reply goal_src/jak1/levels/rolling/rolling-robber.gc:63]: I don't but it didnt hurt to leave it, I can remove if you want.

Comment (Zedb0T) [reply goal_src/jak1/levels/rolling/rolling-robber.gc:63]: It was left in in my testing cycle

Comment (dallmeyer) [reply goal_src/jak1/levels/rolling/rolling-robber.gc:63]: meh I see its only used in rolling-robber code, should be okay

**Pull request merged**